akka-http-core/src/test/scala/akka/http/impl/engine/client/TlsEndpointVerificationSpec.scala

```diff
@@ -47,6 +51,12 @@ class TlsEndpointVerificationSpec extends AkkaSpec("""
         e shouldBe an[Exception]
```

Comment:
That's something I'd need some input from someone of you guys // cc @jrudolph @wsargent @rkuhn